samples/bluetooth/iso_time_sync/README.rst

```diff
@@ -0,0 +1,377 @@
+  * The SDU is transmitted on all the established isochronous channels with the same timestamp.
+    This ensures that all the receivers can toggle their corresponding LEDs at the same time.
+    The very first SDU is provided without a timestamp because the timestamp is not known at this point in time.
+  * **LED1** is configured to toggle synchronously will the **LED1** on all the receivers.
+    The toggle time is determined by the TX timestamp and how it is defined relative to the synchronization reference on the receiver.
+  * If the :kconfig:option:`LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE` Kconfig option is enabled, **LED2** is toggled right before sending the SDU.
+    This can be used to observe the end-to-end latency.
+
+``timed_led_toggle.c``
+
+  This file implements timed toggling a LED.
+  The `GPIOTE` peripheral is used together with `PPI` to achieve microsecond-accurate timing.
+
+``controller_time_<device>.c``
+  The SDU timestamps send and received to the controller are based upon the controller clock.
+  These files allow the application to read the current timestamp and set up a PPI trigger at a given point in time.
+
+  The implementation for nRF52 Series and nRF53 Series are implemented by shadowing an RTC peripheral combined with a TIMER peripheral.
+  The implementation for nRF54L Series uses the GRTC and is much simpler to use.
+
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/iso_time_sync`
+
+.. include:: /includes/build_and_run.txt
+
+
+Testing
+*******
+
+The sample can either demonstrate broadcast isochronous streams or connected isochronous streams.
+
+Testing broadcast isochronous streams
+=====================================
+
+After programming the sample to the development kits, test it by performing the following steps:
+
+1. |connect_terminal_specific|
+#. Reset the kits.
+#. Configure one of the devices as a isochronous broadcaster by typing either ``b`` or ``c`` in the terminal emulator.
+#. Configure the retransmission number and maximum transport latency.
+#. Observe that the broadcaster starts and starts transmitting SDUs.
+#. On the other terminal(s), type ``r`` to configure the device(s) as receivers of the broadcast isochronous stream.
+#. Select which BIS the receiver should synchronize to.
+#. Observe that the device(s) synchronize to the broadcaster and start receiving isochronous data.
+#. Press **BUTTON1** on the broadcaster.
+#. Observe that **LED1** toggles on both the broadcaster and the receivers.
+
+Testing connected isochronous streams
+=====================================
+
+After programming the sample to the development kits, test it by performing the following steps:
+
+1. |connect_terminal_specific|
+#. Reset the kits.
+#. Configure one of the devices as a connected isochronous stream central by typing ``c`` in the terminal emulator.
+#. Configure the retransmission number and the maximum transport latency.
```

Comment:
```suggestion
#. Configure the number of retransmissions and the maximum transport latency.
```
Not so sure about this. Is it the number of retransmission (count) or does a retransmission have a certain number?